src/Santa.java

```diff
@@ -0,0 +1,115 @@
+import java.util.List;
+
+/**
+ * Created by Mitch Messerly on 12/6/2016.
+ */
+public class Santa extends Character {
+    private String suitColor;
+    private int numOfPresents;
+    private int numOfElves;
+    private int energy;
+
+    public Santa(String santaName, String suitColor, int numOfPresents, int numOfElves) {
+        super(santaName);
+        this.suitColor = suitColor;
+        this.numOfPresents = numOfPresents;
+        this.numOfElves = numOfElves;
+        energy = 100;
+    }
+
+    public String getSuitColor() {
+        return suitColor;
+    }
+
+    public int getNumOfPresents() {
+        return numOfPresents;
+    }
+
+    public int getNumOfElves() {
+        return numOfElves;
+    }
+
+    public int getEnergy() {
+        return energy;
+    }
+
+    public void attack(Character target) {
+        double explodeChance = Math.random();
+        if (energy < 10) {
+            System.out.println(getName() + "is too tired, so they fall asleep");
+            try {
+                Thread.sleep(10000);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+            System.out.println(getName() + "woke up with 10 more energy");
+            energy += 10;
+            return;
+        }
+        if (numOfPresents <= 0) {
+            System.out.println(getName() + "doesn't have any presents to throw!");
+            return;
+        }
+        if (target.health >= 100) {
+            System.out.println(target.getName() + "already has full health! There were no effects");
+        } else {
+            if (explodeChance > .85) {
+                System.out.println(getName() + "threw a present at " + target.getName()
+                        + ", and it exploded green healing goo! 30 health was added!");
+                if (target.health > 70) {
+                    target.health = 100;
+                } else {
+                    target.health += 30;
+                }
```

Comment:
Nice edge case checking! Although I might do it like
```java
target.health += 30;
if (target.health > 100) {
    target.health = 100;
}
```
so you don't have to worry about updating the `70` if you update the `30`